src/Invoice.php

```diff
@@ -234,7 +234,7 @@ protected function formatAmount($amount)
     public function view(array $data)
     {
         return View::make('cashier::receipt', array_merge(
-            $data, ['invoice' => $this, 'user' => $this->user]
+            $data, ['invoice' => $this, 'owner' => $this->owner]
```

Comment:
You're right. In case of manual changes to the invoice, developers still referring to the `user` variable will have issues. Fixed that.